models/run.py

```diff
@@ -3,40 +3,132 @@
 sys.path.append(os.getcwd())
 import pickle
 import json
-from base import Event
+import zipfile
+import argparse
 import subprocess
 import numpy as np
 import operator
+import datetime
+import time
+from tqdm import tqdm
 from collections import defaultdict
+from base import Event
 from dummy_human import DummyHuman
```

Comment:
`from models.dummy_human import DummyHuman` will allow running tests from the root run.py

Reply:
ok -- changed!

Reply:
updated.